recipes/slurmpter/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "slurmpter" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: a4dd1eeb5692ce2e92847fb01e73430323c912c3132d53418f0bd1f0ea5dcd34
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
```

Comment:
Under `host` you only need `pip` and `python`. The other packages do not appear to be used directly by `setup.py` or indirectly due to an import in that file. 

Also under both `host` and `run` changes `python` to `python >=3.5`

Reply:
Fixed. Thanks.